lib/api/protocol/pressKeyCode.js

```diff
@@ -0,0 +1,42 @@
+const ProtocolAction = require('./_base-action.js');
```

Comment:
maybe we should have all Appium related commands under an `.appium` namespace? Same as we have `.firefox` and `.chrome` namespaces.

Reply:
I'm okay with putting all the Appium-specific commands under `.appium` namespace, this will provide good feedback to the users that all these commands are only for Appium and won't work with normal browser testing. 

@AutomatedTester @vishalshah133 thoughts?

Reply:
Also, if we were to do that, where would we put the code for all the commands? For `.chrome` and `.firefox` commands, it seems that those commands are already available in `browser.driver` so we didn't implement them ourselves.

Reply:
can be put in an `appium` subfolder inside `api/client-commands` for instance.

Reply:
I'll be working on this next (moving commands to `.appium` namespace) once I complete writing tests for Nightwatch and `create-nightwatch` PRs.